packages/datatrak-web-server/src/routes/TaskMetricsRoute.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
+ */
+
+import { Request } from 'express';
+import { Route } from '@tupaia/server-boilerplate';
+import { getOffsetForTimezone } from '@tupaia/utils';
+import { DatatrakWebTaskMetricsRequest, TaskStatus } from '@tupaia/types';
+import { JOIN_TYPES, QUERY_CONJUNCTIONS, RECORDS } from '@tupaia/database';
+import { assertNewExpression } from '@babel/types';
+
+export type TaskMetricsRequest = Request<
+  DatatrakWebTaskMetricsRequest.Params,
+  DatatrakWebTaskMetricsRequest.ResBody,
+  DatatrakWebTaskMetricsRequest.ReqBody,
+  DatatrakWebTaskMetricsRequest.ReqQuery
+>;
+
+export class TaskMetricsRoute extends Route<TaskMetricsRequest> {
+  public async buildResponse() {
+    const { params, models } = this.req;
+    const { projectId } = params;
+
+    const unassignedTasks = await models.task.count(
+      {
+        [QUERY_CONJUNCTIONS.RAW]: {
+          sql: `(survey.project_id = ? AND assignee_id IS NULL)`,
+          parameters: [projectId],
+        },
+      },
+      { joinWith: RECORDS.SURVEY, joinCondition: ['survey.id', 'task.survey_id'] },
+    );
+
+    const overdueTasks = await models.task.count(
+      {
+        [QUERY_CONJUNCTIONS.RAW]: {
+          sql: `(survey.project_id = ? AND due_date <= ?)`,
+          parameters: [projectId, new Date().getTime()],
+        },
+      },
+      { joinWith: RECORDS.SURVEY, joinCondition: ['survey.id', 'task.survey_id'] },
+    );
+
+    const onTimeCompletionTasks = await models.task.find(
```

Comment:
```suggestion
    const completedTasks = await models.task.find(
```

This is actually just the completed tasks so we should rename it accordingly